### PR TITLE
HandsOn Lab enhancements

### PR DESCRIPTION
**Pull request by robinmanuelthiel**
**Changes:**
- Some Markdown beauty fixes
- Rename `busy` to `isBusy`
- Add `BindingContext` part to first run in Markdown to enhance participant's
understandung of what's going on.

### CODE REVIEW

File: HandsOnLab/README.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ private void OnPropertyChanged([CallerMemberName] string name = null) =>
 private void OnPropertyChanged([CallerMemberName] string name = null)
 {
     var changed = PropertyChanged;
+    
+    if (changed == null)
```
Comment (bartlannoeye) [inline HandsOnLab/README.md]: Since C# 6 this can be a oneliner:
changed?.Invoke(this, new PropertyChangedEventArgs(name));

**Comment by jamesmontemagno** [comment]
Agreed with that above, go ahead and modify it a bit more and i will pull in

File: HandsOnLab/README.md (hunk 1 above)
Comment (robinmanuelthiel) [inline HandsOnLab/README.md]: Sure, but this is an explicit C# 5 Sample for Visual Studio 2012 and 2013. The C# 6 sample right above this section uses the `?.` operator.

**Comment by robinmanuelthiel** [comment]
Hi James. I think we should leave the old C# 5 variant here, as it is an explicit C# 5 sample, that you gave here. I just changed the indentation...

**Pull request merged**